### PR TITLE
Issue 4962 - Fix various UI bugs - Settings and Monitor

### PR DESCRIPTION
**Pull request by droideck**
Description:

Bug 2014924 - Cockpit UI: UX Bugs and other cockpit GUI related defects
Bug 2017441 - cockpit : Export changelog allows to check both 'Export to LDIF For Debugging' options but only takes one into account
Bug 2018101 - cockpit : impossible to create credentials or aliases for replication monitoring synchronization report
Bug 2021250 - cockpit : logging setting entered values for rotation and deletion policies should be checked
Bug 2021278 - Cockpit UI: Unable to Edit Attributes without first searching for the attribute to edit

Related: https://github.com/389ds/389-ds-base/issues/5001

Reviewed by: ?

### CODE REVIEW

File: src/cockpit/389-console/src/dsModals.jsx [hunk 1]
```diff
@@ -569,12 +569,12 @@ export class CreateInstanceModal extends React.Component {
...
                                 />
                                 <FormHelperText isError isHidden={!errObj.createDMPasswordConfirm}>
-                                    Confirmation password must be set and it must match the first password
+                                    Confirmation password must be set and it must match the first password.
                                 </FormHelperText>
                             </GridItem>
                         </Grid>
                         <hr />
-                        <Grid title="Create the Database">
+                        <Grid title="Create a database during the instalation.">
```
Comment (mreynolds389) [inline src/cockpit/389-console/src/dsModals.jsx]: typo, should be "installation"

**Comment by mreynolds389** [comment]
You changed all the "Create New Log Every ..." and "Log File is Older Than ..." settings appear to be a mix of a dropdown and NumberInputs

Screenshot from 2021-11-20 10-20-27

**Comment by mreynolds389** [comment]
In replication monitor, "Add Replication Credntials", the port number is not set, it should have a default value.  And when I increment the number I get a console warning:

checkPropTypes.js:20 Warning: Failed prop type: Invalid prop `port` of type `number` supplied to `ReportCredentialsModal`, expected `string`.
    in ReportCredentialsModal (created by ReplMonitor)

**Comment by mreynolds389** [comment]
Also, not sure if it was always broken, but if the root DN passwords do not match the TextInput is not set to correct validated state.  It never turns red when the passwords don't match

**Comment by droideck** [comment]
> You changed all the "Create New Log Every ..." and "Log File is Older Than ..." settings appear to be a mix of a dropdown and NumberInputs
> 
> Screenshot from 2021-11-20 10-20-27

This part is really weird. It looks okay in full-width mode, but when you shrink the screen - it merges like on your picture...
I may need to rethink the design here as it seems that PF4 fails to Grid properly such a chain of elements (label + numberinput + select on the same line).

**Comment by mreynolds389** [comment]
> This part is really weird. It looks okay in full-width mode, but when you shrink the screen - it merges like on your picture...
> I may need to rethink the design here as it seems that PF4 fails to Grid properly such a chain of elements (label + numberinput + select on the same line).

I think the issue is that the GridItem span's do not add up to 12.  If you reduce the size in this case, thinks do get goofy.  Maybe try setting the very last gridItem's span so that it adds up to 12 total for the Grid.  That might not be enough in all cases and screen sizes, in that case you probably have to equally spread out the spans (it still needs to total 12).

**Comment by mreynolds389** [comment]
Ack!

**Pull request merged**